.github/workflows/docs.yml

```
@@ -0,0 +1,35 @@
+name: Documentation
+
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
```

Let's avoid running this unless we need to, and lets avoid running two sets of it if dependabot or pre-commit pushes to a branch in order to make a PR.

```suggestion
  pull_request:
    paths:
      - "docs/**"
      - "**/docs.yaml"
  push:
    paths:
      - "docs/**"
      - "**/docs.yaml"
    branches-ignore:
      - "dependabot/**"
      - "pre-commit-ci-update-config"
  workflow_dispatch:
```